development-tools/clinic.rst

```diff
@@ -2001,6 +2001,43 @@ The generated glue code looks like this:
 .. versionadded:: 3.13
 
 
+.. _clinic-howto-getter:
+
+How to generate a getter
+------------------------
+
+You can use the ``@getter`` directive to generate an "impl" function
+to define a `getter <https://docs.python.org/3/c-api/structures.html#c.PyGetSetDef.get>`_.
```

Comment:
Hm, when I follow the link, I get a feeling we shouldn't link to the `getter` specifically, but rather the whole `PyGetSetDef`. For someone unfamiliar with getters and setters, it might be confusing to arrive at the minimalistic docs of the `get` member :)

```suggestion
to define a `getter <https://docs.python.org/3/c-api/structures.html#c.PyGetSetDef>`_.
```